src/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/IERS/Finals2000A.cpp

```diff
@@ -450,7 +442,8 @@ Finals2000A Finals2000A::Load(const fs::File& aFile)
 }
 
 Finals2000A::Finals2000A()
-    : span_(Interval::Undefined()),
+    : lastModifiedTimestamp_(Instant::Undefined()),
+      span_(Interval::Undefined()),
       data_(Map<Real, Finals2000A::Data>())
 {
 }
```

Comment:
Seems that we return raw pointers here instead of references? Could be worth fixing just to be safer

Reply:
I agree about this. Want to avoid a [chestertons fence](https://fs.blog/chestertons-fence/) and not change too many things at once. Cool if this is in a follow-up?

Reply:
yup!